### PR TITLE
[BrokenLinksH2] Updated broken link in integrate-microsoft-365-defender-secops-use-cases.md

### PR DESCRIPTION
**Pull request by Angelines-YG**
Fixed broken link for: Microsoft 365 Device Inventory 
Replaced with: /microsoft-365/security/defender-endpoint/device-discovery?view=o365-worldwide

### CODE REVIEW

File: microsoft-365/security/defender/integrate-microsoft-365-defender-secops-use-cases.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ Additionally, the SOC teams could have made the discoveries outlined in the tabl
...
+|SOC Oversight|All assets connected to approved networks are identified and categorized|SOC Oversight, BU owners, application owners, IT asset owners, etc.|Centralized asset management system to discover and list asset category and attributes based on risk.|ServiceNow or other assets. <br><br>[Microsoft 365 Device Inventory](/microsoft-365/security/defender-endpoint/device-discovery?view=o365-worldwide)|Only 70% of assets have been discovered. Microsoft 365 Defender remediation tracking only effective for known assets|Mature asset lifecycle management services to ensure Microsoft 365 Defender has 100% coverage|N|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/integrate-microsoft-365-defender-secops-use-cases.md]:
```suggestion
|SOC Oversight|All assets connected to approved networks are identified and categorized|SOC Oversight, BU owners, application owners, IT asset owners, etc.|Centralized asset management system to discover and list asset category and attributes based on risk.|ServiceNow or other assets. <br><br>[Microsoft 365 Device Inventory](/microsoft-365/security/defender-endpoint/device-discovery)|Only 70% of assets have been discovered. Microsoft 365 Defender remediation tracking only effective for known assets|Mature asset lifecycle management services to ensure Microsoft 365 Defender has 100% coverage|N|
```

Comment (JohanFreelancer9) [inline microsoft-365/security/defender/integrate-microsoft-365-defender-secops-use-cases.md]:
```suggestion
|SOC Oversight|All assets connected to approved networks are identified and categorized|SOC Oversight, BU owners, application owners, IT asset owners, etc.|Centralized asset management system to discover and list asset category and attributes based on risk.|ServiceNow or other assets. <br><br>[Microsoft 365 Device Inventory](/microsoft-365/security/defender-endpoint/device-discovery)|Only 70% of assets have been discovered. Microsoft 365 Defender remediation tracking only effective for known assets|Mature asset lifecycle management services to ensure Microsoft 365 Defender has 100% coverage|N|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Angelines-YG, kindly commit the suggested change. Many thanks.

**Comment by Angelines-YG** [comment]
@JohanFreelancer9, @officedocspr2

The Content & Learning team is fixing broken links on docs.microsoft.com for the rest of H2. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only link fixes and does not change other content. Please review within five business days and merge, or comment in the PR with any changes you'd like to see. Thanks!

**Comment by Angelines-YG** [comment]
@JohanFreelancer9 

Suggested change has been committed. Please let me know if there is anything that needs to be changed. 
The Content & Learning team is fixing broken links on docs.microsoft.com for the rest of H2. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only link fixes and does not change other content. Please review within five business days and merge, or comment in the PR with any changes you'd like to see. Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]
Thank you @Angelines-YG.

**Review by denisebmsft — APPROVED** [review]
Thank you @Angelines-YG @JohanFreelancer9

**Pull request merged**